config/config/common-config.yaml

```diff
@@ -158,6 +158,19 @@ prometheus:
   tolerations: []
   affinity: {}
   nodeSelector: {}
+  ## Predictive alert if the resource usage will hit the set percentage in 3 days
+  predictLinear:
+    enabled: true
+    disklimit: 66
+    usagelimit: 95
+    ## for each cpu and memory add the node pattern for which you want to create an alert and the limit value
+    requestlimit:
+      cpu:
+        worker: 80
+        elastisys: 80
+      memory:
+        worker: 80
+        elastisys: 80
```

Comment:
You could move `elastisys` to wc.

Reply:
if I move it I think I will exclude it from our alerts

Reply:
Right, because the alerts are evaluated in sc. Didn't think of that.